src/com/jwetherell/algorithms/numbers/Integers.java

```diff
@@ -8,6 +8,12 @@ public class Integers {
 
     private static final BigDecimal ZERO = new BigDecimal(0);
     private static final BigDecimal TWO = new BigDecimal(2);
+    
+    private static final int BILLION = 1000000000;
+    private static final int MILLION = 1000000;
+    private static final int THOUSAND = 1000;
+    private static final int HUNDRED = 100;
+    private static final int TEN = 10;
```

Comment:
This change makes sense but you should comment it
```suggestion
    // Constants are used to reduce mistyping later in the code
    private static final int BILLION = 1000000000;
    private static final int MILLION = 1000000;
    private static final int THOUSAND = 1000;
    private static final int HUNDRED = 100;
    private static final int TEN = 10;
```